nextcord/interactions.py

```diff
@@ -1113,3 +1114,79 @@ async def delete(self, *, delay: Optional[float] = None) -> None:
         """
 
         await self._state._interaction.delete_original_message(delay=delay)
+
+
+class PartialInteractionMessage(_InteractionMessageMixin, Hashable):
+    """Represents the original interaction response message when only the
+    application state and interaction token are available.
+
+    This allows you to edit or delete the message associated with
+    the interaction response. This object is returned when responding to
+    an interaction with :meth:`InteractionResponse.send_message`.
+
+    This does not support most attributes and methods of :class:`nextcord.Message`.
+    The :meth:`~PartialInteractionMessage.fetch` method can be used to
+    retrieve the full :class:`InteractionMessage` object.
+
+    .. versionadded:: 2.0
+    """
+
+    def __init__(self, state: _InteractionMessageState):
+        self._state = state
+
+    async def fetch(self) -> InteractionMessage:
+        """|coro|
+
+        Fetches the original interaction response message associated with the interaction.
+
+        Repeated calls to this will return a cached value.
+
+        Raises
+        -------
+        HTTPException
+            Fetching the original response message failed.
+        ClientException
+            The channel for the message could not be resolved.
+
+        Returns
+        --------
+        InteractionMessage
+            The original interaction response message.
+        """
+        return await self._state._interaction.original_message()
+
+    @property
+    def author(self) -> Optional[ClientUser]:
+        """Optional[:class:`ClientUser`]: The client user that responded to the interaction."""
+        return self._state._interaction.client.user
+
+    @property
+    def channel(self) -> Optional[InteractionChannel]:
+        """Optional[Union[:class:`abc.GuildChannel`, :class:`PartialMessageable`, :class:`Thread`]]: The channel the interaction was sent from.
+
+        Note that due to a Discord limitation, DM channels are not resolved since there is
+        no data to complete them. These are :class:`PartialMessageable` instead.
+        """
+        return self._state._interaction.channel
+
+    @property
+    def guild(self) -> Optional[Guild]:
+        """Optional[:class:`Guild`]: The guild the interaction was sent from."""
+        return self._state._interaction.guild
+
+    def __repr__(self):
+        return f"<{self.__class__.__name__} author={self.author!r} channel={self.channel!r} guild={self.guild!r}>"
```

Comment:
Should there be commas between args maybe?

Reply:
It seems like the other objects in Nextcord do not have commas, so I think this is the way it is supposed to be